candle-nn/src/linear.rs

```diff
@@ -19,6 +19,7 @@
 //! ```
 use candle::{Result, Tensor};
 
+/// See the documentation for [`mod@super::linear`] for more information.
```

Comment:
Rendered:

![image](https://github.com/huggingface/candle/assets/1732289/623f535f-f574-4347-9809-f7254cafa3d5)

I didn't want to duplicate the module docs, it's not great, but it helps people find the actual docs.